apache/_docroot.shtml

```diff
@@ -1,29 +1,28 @@
 <div class="section">
     <h2>Can Tugboat find your document root?</h2>
     <p>
-        Tugboat looks for a folder named
-        <tt>docroot</tt> in your repository to serve content from. This is a common pattern, but many projects have their own
-        preference. If your document root is in a different location you have a couple of different options.
+        Tugboat looks for a folder named <tt>docroot</tt> in your repository to serve content from. This is a 
+        common pattern, but many projects have their own preference. If your document root is in a different location 
+        you have a couple of different options.
     </p>
     <h3>Option 1. Add a <tt>docroot/</tt> folder to your repository</h3>
     <p>
       This is the most straightforward fix for Tugboat, but can be problematic if your repository has already been 
-      deployed elsewhere. If you choose this option, create a <tt>docroot</tt> folder and move your existing content 
-      you want Apache to serve into that folder. Be sure to follow up on other deployments to make sure things still 
-      work.
+      deployed elsewhere. If you choose this option, create a <tt>docroot</tt> folder for Tugboat to look in. If your 
```

Comment:
Simply creating a docroot folder is not enough. The repo's web content needs to be moved into that folder.